quantumdb-core/src/main/java/io/quantumdb/core/migration/operations/AlterColumnMigrator.java

```diff
@@ -73,13 +73,8 @@ private String normalize(String input) {
 			char first = trimmed.charAt(0);
 			char last = trimmed.charAt(trimmed.length() - 1);
 
-			if (first == last) {
-				if (first == '\'') {
```

Comment:
Is there a reason why you want these to be in another if statement?

Reply:
This is just to avoid having to check `first == last` for every subsequent if statement. But I'm ok with this change 👍